.github/ISSUE_TEMPLATE/direct-from-marketing.yml

```diff
@@ -0,0 +1,43 @@
+name: Add marketing blog content
+description: >
+  For adding marketing-approved content to the DevBlog.
+labels: ["content","developer blog"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Let's get some content up!
+        
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: your preferred email and/or dbt Slack handle
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: from-discussion
+    attributes:
+      label: This issue is being made to create a marketing-approved piece.
+      description: >
+        This content will typically fall under glossary pages, community recap events, and devblog content for campaigns. Please reach out to John Blust if there's any confusion here.
+      options:
+        - label: I am directly creating this issue for a marketing-approved piece.
```

Comment:
Do you perhaps want to make this even more limited? Like, 'I work for dbt Labs and am directly etc etc' -- I'm not sure if there would ever be a reason for somebody outside the company to create this type of issue, and if not, probably good to add for extra clarity.

Reply:
Good callout!